app/src/main/java/com/everest/movieapp/api/RetrofitHelper.kt

```diff
@@ -0,0 +1,14 @@
+package com.everest.movieapp.api
+
+import com.everest.movieapp.constants.Constants.Companion.BASE_URL
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitHelper {
+
+    fun getInstance(): Retrofit {
+        return Retrofit.Builder().baseUrl(BASE_URL)
```

Comment:
this will create new client every time you call getInstance(), we can use single instance